### ISSUE TITLE
Look at needs for upcoming events & figure out template plan

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/plan#683]
Let's look at upcoming events and figure out if/how we can templatize some of the design or programmatic needs. ie. 
* Mozfest
* SourceCon
* Hive Buzz
* Club meetups
cc @edrushka

**Comment by cassiemc** [comment]
Some shared needs might be:
* website
* schedule app
* swag (we should get more specific here)
   * name badges
   * printed guides
   * etc

And can determine which need/should be bespoke, and how to make spinning them up faster.

**Comment by edrushka** [comment]
YES! Can we wait until I get back from Chicago? I'll have a much better handle on things by then. cc @Saallen

**Comment by cassiemc** [comment]
Yes absolutely. No rush!

**Comment by edrushka** [comment]
I'm going to assign myself for now so I remember to book the MozFest chat at the beginning of Feb. I think @lovegushwa and @natalieworth will have some good input on this. @KevZawacki too.

**Comment by edrushka** [comment]
@vazquez will also have some insights on this following Hive Chicago Buzz.

**Comment by vazquez** [comment]
I added some thoughts to this Google Doc. Feel free to add/edit

https://docs.google.com/a/mozilla.com/document/d/1pefmuP6bp2lElMNCna8QB4JwmDlgwaIfAkmn21E0bDw/edit?usp=sharing

**Comment by omnignorant** [comment]
Ha @vazquez I'm reading some very specific learnings from Chicago in that document :wink: :point_right:  :nose:

**Comment by edrushka** [comment]
hehe. Me too ;)

@vazquez can you give me edit or comment permissions for your doc? 
@cassiemc do you think we should meet about this, or should we finalize the requirements in Ricardo's doc, and discuss when we start to plan the next convening?

**Comment by vazquez** [comment]
Done! Anyone with the link can edit: https://docs.google.com/a/mozilla.com/document/d/1pefmuP6bp2lElMNCna8QB4JwmDlgwaIfAkmn21E0bDw/edit?usp=sharing

**Comment by cassiemc** [comment]
@edrushka I have some thoughts about potential additions based on last year's mozfest. Will add, and we should have @lovegushwa and @natalieworth review as well.

**Comment by cassiemc** [comment]
Also, @HPaulJohnson probably has thoughts around typical comms/ promotion needs.

**Comment by edrushka** [comment]
I have added my hopes and dreams to the spreadsheet. Once everyone's had a chance to weigh in, perhaps some of us can meet (at minimum, @cassiemc and I but of course anyone is welcome).

**Comment by edrushka** [comment]
@cassiemc shall I set up a meeting to discuss? Perhaps we could create a prioritized list of templates and talk about timelines and design capacity.

**Comment by cassiemc** [comment]
Sounds good @edrushka

**Comment by cassiemc** [comment]
@edrushka based on our convo today do you think we should apply the ```likely to close``` label?

**Comment by edrushka** [comment]
I think we can close since technically we've looked at our needs and made a plan. Next step is implementation, which will be filed through individual tickets.

For those following this ticket, Cassie and I made a shortlist of items to templatize as a first step: 
- lanyard/name tag
- handout/guidebook
- poster
I will connect with @ldecoursy  to discuss which upcoming Hive event makes the most sense to try these templates at.
